delete.php

```diff
@@ -0,0 +1,26 @@
+<?php
+$db = new PDO('mysql:host=192.168.20.20;dbname=portfolio', 'root', '');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+if (!empty($_GET['id'])) {
+
+    $id = $_GET['id'];
+
+    $sql = "DELETE FROM `portfolio` WHERE `id` = :id;";
```

Comment:
Going to allow this but bear in mind that setting a "deleted" field on each record and changing your SELECT statements to ignore rows that are "deleted" is much safer